core/src/components/select/select.ionic.scss

```diff
@@ -0,0 +1,185 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@use "./select.common";
+@use "./select.ionic.outline" as outline;
```

Comment:
```suggestion
@use "./select.ionic.outline";
```

Reply:
This is necessary to keep else Sass throws this error:
![Screenshot 2024-11-07 at 11 33 11 AM](https://github.com/user-attachments/assets/575b5db9-1a7a-4b6a-b6fb-553552410e01)